src/main/resources/lib/login-util/config.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
```

Comment:
igjen, gitignore :)

Reply:
hehe :)
Jeg lar js filene ligge for nå fordi jeg ikke vil røre for mye med den orginale koden.